main_test.go

```diff
@@ -0,0 +1,427 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/sigstore/cosign/cmd/cosign/cli/fulcio"
+	"github.com/sigstore/cosign/pkg/cosign"
+	"github.com/sigstore/cosign/pkg/oci"
+	"github.com/sigstore/cosign/pkg/oci/layout"
+	"github.com/slsa-framework/slsa-verifier/pkg"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+)
+
+func errCmp(e1, e2 error) bool {
+	return errors.Is(e1, e2) || errors.Is(e2, e1)
+}
+
+func pString(s string) *string {
+	return &s
+}
+
+// Versions of the builders to test.
+var golangGeneratorVersions = []string{"v0.0.2", "v1.1.1"}
+
+func Test_runVerifyGo(t *testing.T) {
+	// t.Parallel()
```

Comment:
remove?

Reply:
@asraa please advise

Reply:
I think your branch may be out of date: https://github.com/slsa-framework/slsa-verifier/blob/77f8bc623b96f6124ff2cc4a1daade743f01c6b3/cli/slsa-verifier/main_test.go#L66

This can be removed and is at HEAD

Reply:
I had a merge problem, you're correct.